apps/chat/src/components/Header/Header.tsx

```diff
@@ -30,6 +33,13 @@ const Header = () => {
     UISelectors.selectIsUserSettingsOpen,
   );
   const isOverlay = useAppSelector(SettingsSelectors.selectIsOverlay);
+  const customLogo = useAppSelector(UISelectors.selectCustomLogo);
+  const enabledFeatures = useAppSelector(
+    SettingsSelectors.selectEnabledFeatures,
+  );
+  const customLogoUrl =
```

Comment:
need to use `encodeApiUrl`

Reply:
Moved usage of **encodeApiUrl** from SettingsDialog here.
`const customLogoUrl =
    enabledFeatures.has(Feature.CustomLogo) &&
    customLogo &&
    `api/${ApiUtils.encodeApiUrl(customLogo)}`;`